photon-lib/src/main/java/org/photonvision/PhotonCamera.java

```diff
@@ -374,13 +374,37 @@ else if (!isConnected()) {
         if (!versionString.isEmpty() && !PhotonVersion.versionMatches(versionString)) {
             // Error on a verified version mismatch
             // But stay silent otherwise
-            DriverStation.reportWarning(
+
+            String bfw = "";
+            bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
+            bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
+            bfw += ">>>    ____  _________    ____     ________  ___________    __   \n";
+            bfw += ">>>   / __ \\/ ____/   |  / __ \\   /_  __/ / / /  _/ ___/   / / \n";
+            bfw += ">>>  / /_/ / __/ / /| | / / / /    / / / /_/ // / \\__ \\   / /  \n";
+            bfw += ">>> / _, _/ /___/ ___ |/ /_/ /    / / / __  // / ___/ /  /_/     \n";
+            bfw += ">>>/_/ |_/_____/_/  |_/_____/    /_/ /_/ /_/___//____/  (_)      \n";
+            bfw += ">>>                                                              \n";
+            bfw += ">>> You are running an incompatible version                      \n";
+            bfw += ">>> of PhotonVision on your coprocessor!                         \n";
+            bfw += ">>>                                                              \n";
+            bfw += ">>> This is neither tested nor supported.                        \n";
+            bfw += ">>> You MUST update either PhotonVision, PhotonLib, or both.     \n";
+            bfw += ">>>                                                              \n";
+            bfw += ">>> Your code will now crash. We hope your day gets better.      \n";
+            bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
+            bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
```

Comment:
Here's an option if JDK 11 compat is needed: https://godbolt.org/z/er4ME3a64

If JDK 11 compat is dropped, multiline comments are in JDK 15 and up: https://stackoverflow.com/a/50155171

Reply:
We'll upgrade to Java 17 eventually so I'll go with the godbolt thing for now